components/Chatbot.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from "react"
+import Script from "next/script"
+declare var Landbot: any
+
+const Chatbot = () => {
+  function initLandbot() {
+    var s = document.createElement("script")
+    s.type = "text/javascript"
+    s.async = true
+    s.addEventListener("load", function () {
+      new Landbot.Popup({
+        configUrl: "https://chats.landbot.io/v3/H-1228722-I7RHH311Y3TAH6H3/index.json",
+      })
+    })
+    s.src = "https://cdn.landbot.io/landbot-3/landbot-3.0.0.js"
+    var x = document.getElementsByTagName("script")[0]
+    if (x.parentNode && typeof x.parentNode === "object") {
+      x.parentNode.insertBefore(s, x)
+    }
+  }
+
+  return (
+    <>
+      <Script
+        id="chatbot"
+        src="https://cdn.landbot.io/landbot-3/landbot-3.0.0.js"
```

Comment:
Looks like we're loading the script twice since we're also injecting a script tag in `initLandbot`

Reply:
I see. on the Script we can remove the 'src' and see if this works. the 'id' can also get removed.

Reply:
turns out we need to inject it twice. It shouldn't be a problem either way.